net.hachune.mikutter.git-source.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Is it really worth it to keep it separately?

Reply:
Yes. This file is imported by two files (net.hachune.mikutter.yaml and net.hachune.mikutter.rubygems-module.json).

Reply:
Ah, okay, I didn't notice.